server/db/models/index.js

```diff
@@ -19,7 +19,12 @@ Applications.belongsTo(Forms, {
 Admins.hasMany(Comments)
 Comments.belongsTo(Admins, {foreignKey: 'adminId', targetKey: 'id'})
 
-Applications.belongsTo(Comments, {
+// Applications.belongsTo(Comments, {
+//   foreignKey: 'applicantName',
+//   targetKey: 'applicantName'
+// })
+
+Comments.belongsTo(Applications, {
```

Comment:
@amesk3 and @ndittren,

I ran into this error when trying to create an application as an applicant:
```
SequelizeForeignKeyConstraintError: insert or update on table 
"applications" violates foreign key constraint "applications_applicantName_fkey"

detail:'Key (applicantName)=(Trey Stevens) is not present in table "comments".',
```

I'm not really sure what to do with foreign keys, so for this portion I just swapped the "Applications" and "Comments" around and it worked. My assumption from the error was that the `Comments` table was not being created? Or at least when an application is created a comments table is then created matching to a `foreign key`, but I'm pretty clueless on how to do all of that. I figured it would be easier if I just get an instance of the application running and then we can debug. Please let me know the proper way to go about this!